.github/workflows/automate-team-review-assignment-config.yml

```diff
@@ -1,13 +1,13 @@
 name: 'Automate assigning team for review.'
-on: 
-  pull_request:
-    types: [ opened ]
+on:
+    pull_request:
+        types: [ready_for_review]
 jobs:
-  add-reviews:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Check config and assign reviews
-        uses: acq688/Request-Reviewer-For-Team-Action@v1.1
-        with:
-          config: ".github/automate-team-review-assignment-config.yml"
-          GITHUB_TOKEN: ${{ secrets.PAT_FOR_ACTIONS }}
+    add-reviews:
+        runs-on: ubuntu-latest
```

Comment:
```suggestion
        if: ! github.event.pull_request.draft
        runs-on: ubuntu-latest
```

Reply:
As I wasn't able to successfully test this PR, I looked up how to run a GitHub Action on a no draft PR and stumbled upon https://stackoverflow.com/questions/68349031/only-run-actions-on-non-draft-pull-request. I suggested the change accordingly.